provisioning/miniprov/hare_mp/store.py

```diff
@@ -141,6 +162,9 @@ def search_val(self, parent_key: str, search_key: str,
         """
         return self.conf.search(self.index, parent_key, search_key, search_val)
 
+    def get_motr_clients(self) -> List[Dict[str, Any]]:
```

Comment:
Please add documentation for this new function.
Thanks!

Reply:
Done